labgrid/autoinstall/main.py

```diff
@@ -73,7 +73,7 @@ def run(self):
             self.context['target'] = self.target
             if self.target is None:
                 raise KeyError
-        except:
+        except KeyError:
```

Comment:
Other exceptions can be generated by env.get_target() during target creation, which should also be caught by this catch all.